contracts/SmartLoan.sol

```diff
@@ -0,0 +1,459 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import "./IAssetsExchange.sol";
+import "./SupportedAssets.sol";
+import "./Pool.sol";
+import "redstone-flash-storage/lib/contracts/message-based/PriceAwareUpgradeable.sol";
+
+
+/**
+ * @title SmartLoan
+ * A contract that is authorised to borrow funds using delegated credit.
+ * It maintains solvency calculating the current value of assets and borrowings.
+ * In case the value of assets held drops below certain level, part of the funds may be forcibly repaid.
+ * It permits only a limited and safe token transfer.
+ *
+ */
+contract SmartLoan is OwnableUpgradeable, PriceAwareUpgradeable, ReentrancyGuardUpgradeable {
+
+  uint256 public constant PERCENTAGE_PRECISION = 1000;
+
+  uint256 public constant LIQUIDATION_BONUS = 100;
+  uint256 private constant LIQUIDATION_CAP = 200;
+
+  uint256 public MAX_LTV = 5000;
+  uint256 public MIN_SELLOUT_LTV = 4000;
```

Comment:
Could we switch it to lowerCase, now it looks like a constant value

Reply:
@kamilovsky01

Reply:
They are constants now. And there is no need to initialize them in initialize.